astropy/wcs/utils.py

```diff
@@ -52,8 +52,8 @@ def add_stokes_axis_to_wcs(wcs, add_before_ind):
 def _wcs_to_celestial_frame_builtin(wcs):
 
     # Import astropy.coordinates here to avoid circular imports
-    from astropy.coordinates import (FK4, FK5, ICRS, ITRS, FK4NoETerms,
-                                     Galactic, SphericalRepresentation)
+    from astropy.coordinates import (
+        FK4, FK5, ICRS, ITRS, FK4NoETerms, Galactic, SphericalRepresentation)
```

Comment:
Some of this will change when `black` is rolled out. The current `isort` configuration is not compatible and one of the steps for `black` will be to run a whole-repository re-isort.